docs/source/api/plugin/usage-reporting.md

```diff
@@ -111,27 +111,37 @@ The only properties of the reported error you can modify are its `message` and i
 </td>
 <td>
 
-This option allows you to choose if Apollo Server should calculate detailed per-field statistics for a particular request. It is only called for executable operations: operations which parse and validate properly and which do not have an unknown operation name. It is not called if an [`includeRequest`](#includerequest) hook is provided and returns false.
+This option enables you to configure whether Apollo Server calculates detailed per-field statistics for each operation. It is used only for operations that parse and validate properly. It is _not_ used if you provide an [`includeRequest`](#includerequest) hook that returns `false`.
```

Comment:
You removed the thing about unknown operation name.  If you provide an operation name alongside a document which does not match an operation in the document then it will fail, but that is not technically a validation failure (validation is a function strictly of document and schema, not of out-of-band operation name or variables).

Reply:
Cool, I tried an alternate approach here in the followup commit for flow